recipes/gnu-time/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gnu-time" %}
+{% set version = "1.7" %}
+{% set sha256 = "e37ea79a253bf85a85ada2f7c632c14e481a5fd262a362f6f4fd58e68601496d" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: time-1.7.tar.gz
+  url: http://mirrors.ocf.berkeley.edu/gnu/time/time-1.7.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
```

Comment:
Does this still make sense if we are using `m2w64-toolchain`? TBH I'm not sure, but am thinking it doesn't.